src/index.js

```diff
@@ -336,7 +337,10 @@ class BeetConnection {
      * @param {String} chain Symbol of the chain to be linked
      * @returns {Promise} Resolves to false if not linked after timeout, or to result of 'link' Beet call
      */
-    async link(chain = null) {
+    async link(chain = null, requestDetails = null) {
+        if (requestDetails == null) {
+            requestDetails = ["account"];
```

Comment:
why? I thought the idea was to allow completely anonymous usage as well

Reply:
Yes, this is reflecting what I think is the default usage. Anonymous would be `[ ]`